driver_cpl/driver/prep_ocn_mod.F90

```diff
@@ -1192,18 +1198,26 @@ subroutine prep_ocn_calc_r2x_ox(timer)
     !
     ! Local Variables
     integer :: eri
-    type(mct_avect), pointer :: r2x_rx
-    character(*), parameter  :: subname = '(prep_ocn_calc_r2x_ox)'
+    type(mct_avect), pointer   :: r2x_rx
+    character(len=shr_kind_cs) :: fldlistl, fldlisti
+    character(*), parameter    :: subname = '(prep_ocn_calc_r2x_ox)'
     !---------------------------------------------------------------
 
     call t_drvstartf (trim(timer),barrier=mpicom_CPLID)
+    if (lwiso_runoff) then
+      fldlistl='Forr_rofl:Forr_rofl_16O:Forr_rofl_18O:Forr_rofl_HDO'
```

Comment:
This hard-coding of field lists feels very fragile to me. But I guess that's not your fault. I'm fine with the change you made here, given that this list was already hard-coded.